lib/types.js

```diff
@@ -117,7 +117,7 @@ function generateTypeConversion(ctx, name, idlType, argAttrs = [], parentName, e
     // string or number type compatible with webidl-conversions
     generateGeneric(`conversions["${idlType.idlType}"]`);
   } else if (resolvedTypes.has(ctx.typeOf(idlType.idlType))) {
-    // dictionaries, enumerations, and interfaces
+    // callbacks, dictionaries, enumerations, and interfaces
```

Comment:
```suggestion
    // callback interfaces, dictionaries, enumerations, and interfaces
```

You should also delete the `ctx.typeOf(idlType.idlType) === "callback interface"` block that comes afterwards.